inst/templates/wercker.yml

```diff
@@ -0,0 +1,5 @@
+box: rocker/hadleyverse
```

Comment:
So far I've built upon Dirk and Carl's efforts, but we might be able to squeeze it just a bit further. Note that my image includes `devtools` and all system deps necessary for it. (It also includes R-devel.)